tests/test_conflicts.py

```diff
@@ -182,57 +188,63 @@ def test_list_conflicts(data_archive, cli_runner):
         )
 
         expected_geojson = {
+            "type": "FeatureCollection",
```

Comment:
You've rearranged this expected_geojson a bit but it's not obvious to me what's changed.
Has anything changed?
Is one of these orders the "right" order? I guess it would be good to have it in the order that the code actually prints it... would make the test easier to read...

Reply:
> I guess it would be good to have it in the order that the code actually prints it

Rearranged according to this.

Reply:
Did anything change?